app/models/media_attachment.rb

```diff
@@ -22,14 +22,16 @@
 class MediaAttachment < ApplicationRecord
   self.inheritance_column = nil
 
-  enum type: [:image, :gifv, :video, :unknown]
+  enum type: %i(image gifv video unknown audio)
 
-  IMAGE_FILE_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif'].freeze
-  VIDEO_FILE_EXTENSIONS = ['.webm', '.mp4', '.m4v', '.mov'].freeze
+  IMAGE_FILE_EXTENSIONS = %w(.jpg .jpeg .png .gif).freeze
+  VIDEO_FILE_EXTENSIONS = %w(.webm .mp4 .m4v .mov).freeze
+  AUDIO_FILE_EXTENSIONS = %w(.ogg .oga .mp3).freeze
```

Comment:
What about adding support for FLAC and WAV files?

Reply:
Also .opus would be appreciated!

Reply:
https://caniuse.com/#feat=flac
https://caniuse.com/#search=wav

Can't see why not.

Reply:
See, this is the can of worms I wanted to avoid by not messing with audio. FLAC and WAV are fucking huge, and converting them takes precious CPU cycles. Makes me question whether this whole PR is worth it.

Reply:
To be honest, we already convert videos, which is probably way more CPU-intensive than converting audio files.

Reply:
I would say the people who are uploading audio files already know how to convert them. Anything else will mess up the code even more. This feature is nice and I would really like to get it merged. Everyone else who needs FLACs, WAVs or OPUS can upload theirs to Pleroma or whatever. That's why these things exist.

Reply:
@ThibG For reference we convert those because we have no choice in regards to mobile devices. But no urgent need exists for audio.